dask/dataframe/multi.py

```diff
@@ -1013,6 +1019,9 @@ def concat(dfs, axis=0, join="outer", interleave_partitions=False):
     interleave_partitions : bool, default False
         Whether to concatenate DataFrames ignoring its order. If True, every
         divisions are concatenated each by each.
+    ignore_unknown_divisions: boolean, default False
+        Whether to warn when concatenating dask series/dataframes
+        with unknown divisions. If True the warning won't be emitted.
```

Comment:
This option is the opposite, whether to *not* warn:

```suggestion
        By default a warning is raised if any input has unknown divisions.
        Set to True to disable this warning.
```